source/Calamari.AzureWebApp/Calamari.AzureWebApp.csproj

```diff
@@ -5,7 +5,7 @@
         <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
         <IsPackable>false</IsPackable>
         <OutputType>Exe</OutputType>
-        <RuntimeIdentifiers>win-x64;linux-x64;osx-x64;linux-arm;linux-arm64</RuntimeIdentifiers>
+        <RuntimeIdentifiers>win-x64;win7-x86;linux-x64;osx-x64;linux-arm;linux-arm64</RuntimeIdentifiers>
```

Comment:
No idea why i need it!

Reply:
Because you had `win7-x86` on the `Calamari.Azure.Scripting` project, maybe remove it from there